### ISSUE TITLE
WPS204: Overused expression should not count unary minus operator

### ISSUE THREAD

**Issue by pawelarybak** [wemake-services/wemake-python-styleguide#1753]
# Bug report

Using unary minus operator often breaks rule WPS204

## What's wrong

Using unary minus operator even with named variable violates rule WPS204.
Example:
```python
my_value = 5
position = (0, -my_value)
```

## How it should be

No violation should be raised.

### flake8 information

Contents of `flake8 --bug-report`:

### pip information

Contents of `pip freeze`:

**Comment by sobolevn** [comment]
@pawelarybak are you interested in providing a fix?

**Comment by pawelarybak** [comment]
I can give it a shot.

**Comment by sobolevn** [comment]
Thanks! @pawelarybak
